src/api/dimensions.js

```diff
@@ -99,6 +100,28 @@ export const dataElementGroupsQuery = {
     }),
 }
 
+export const apiFetchItemsByDimension = ({
```

Comment:
This is ok for now, but I would like it to follow the same format as the other queries here, if the query definition and fetch function stays here.
The other fetch functions return the data instead of the query object, so the consumer doesn't have to know anything about the query definition or the namespace used.

Reply:
Done